web-nfc/NDEFWriter_push.https.html

```diff
@@ -292,6 +292,15 @@
   assertNDEFMessagesEqual(buffer_view, mockNFC.pushedMessage());
 }, "Test that NDEFWriter.push succeeds when message is ArrayBufferView.");
 
+nfc_test(async (t, mockNFC) => {
```

Comment:
Rather than adding this case here, I think refining 'NDEFRecord constructor with external record type' in NDEFRecord_constructor.https.html is better. The error happens because we missed the support for constructing an external type NDEFRecord correctly with the input ArrayBufferView, not due to push() logic.

Reply:
Thanks, done.